backend/scrumblr-api/.aws-sam/build.toml

```diff
@@ -1,8 +1,8 @@
 # This file is auto generated by SAM CLI build command
```

Comment:
Please don't commit this folder next time. The content of this directory are the artefacts outputted from your sam build command, and can always be recreated. Add this folder to .gitignore